<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>josef ling</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>1005</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drop Data Collection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/downarrow.png</normaloff>:/images/downarrow.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QwtPlot" name="qwtPlot_1">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="canvasBackground" stdset="0">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>46</red>
           <green>47</green>
           <blue>48</blue>
          </color>
         </brush>
        </property>
        <property name="autoReplot" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtPlot" name="qwtPlot_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Experiment"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuPlot_Settings">
     <property name="title">
      <string>Plot Settings</string>
     </property>
     <widget class="QMenu" name="menuNew_Plot">
      <property name="title">
       <string>New Plot ...</string>
      </property>
      <addaction name="actionOverlays_Existing"/>
      <addaction name="actionReplaces_Existing"/>
     </widget>
     <addaction name="menuNew_Plot"/>
    </widget>
    <addaction name="actionSensor_Settings"/>
    <addaction name="menuPlot_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartCollection"/>
   <addaction name="actionClearPlot"/>
   <addaction name="separator"/>
   <addaction name="actionHome"/>
   <addaction name="actionZoom"/>
   <addaction name="actionPan"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStartCollection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/collect_off.png</normaloff>
     <normalon>:/images/collect_on.png</normalon>:/images/collect_off.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start  Data Collection</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionClearPlot">
   <property name="text">
    <string>Clear Plot</string>
   </property>
   <property name="toolTip">
    <string>Clear all plots</string>
   </property>
  </action>
  <action name="actionSensor_Settings">
   <property name="text">
    <string>Sensor Settings</string>
   </property>
  </action>
  <action name="actionNew_Plot_Overlays">
   <property name="text">
    <string>New Plot ...</string>
   </property>
   <property name="toolTip">
    <string>New Plot ...</string>
   </property>
  </action>
  <action name="actionOverlays_Existing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Overlays Existing</string>
   </property>
  </action>
  <action name="actionReplaces_Existing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Replaces Existing</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedoff>:/images/zoom_off.png</selectedoff>
     <selectedon>:/images/zoom_on.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedoff>:/images/pan_off.png</selectedoff>
     <selectedon>:/images/pan_on.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionNew_Experiment">
   <property name="text">
    <string>New Experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
